tests/optimizer/transformers/data/expected_prophetnet_output_no_cache.hypo

```diff
@@ -0,0 +1,16 @@
+[SEP]'' it's a very disturbing scene.'' he added, `'so far,'' the prosecutor said, adding, ` ` so far... so far'' said the prosecutor, who told cnn on wednesday that he is not aware of any such video.'he said. the prosecutor also said that he has not seen any videos, he said, but that the person who has such a video is not being aware of the video. the publication said that the video was recovered at the site, but he said that was not the case. the reports by the two publications said that they are not. the publications said they did not have the video, and the two magazines said that neither of the publications had the video and that the publications did not know that the videos were found. the magazine said that no such video was found. asked that the report is not a video, the publications told cnn. [SEP] [PAD] [PAD] [PAD] [PAD]
```

Comment:
What are the differences between this file and the above one?

Reply:
This is vestigial from some debug/sanity checking I was doing. I will remove it. Essentially, prophetnet has slightly different outputs depending on if it is with or without cache which I was checking, but since it takes 30 mins to run the test without cache on all 128 samples, this file is shorter so I could run it faster.

Reply:
make sense